src/PSR7/SpecFinder.php

```diff
@@ -244,7 +246,9 @@ public function findHeaderSpecs(OperationAddress $addr): array
         // 1. Collect operation level headers from "parameters" keyword
         // An API call may require that custom headers be sent with an HTTP request. OpenAPI lets you define custom
         // request headers as in: header parameters.
+        /** @var array<string, Header> $headerSpecs */
```

Comment:
this is invalid according to later usage and return type. you cannot put `Parameter` into `array<string, Header>`. So it should be same as return type right here

Reply:
Fixed.